src/main/java/hudson/plugins/ansicolor/AnsiOutputStream.java

```diff
@@ -42,7 +42,7 @@
  */
 public class AnsiOutputStream extends FilterOutputStream {
 
-    public static final byte[] REST_CODE = resetCode();
+    private static final byte[] REST_CODE = resetCode();
```

Comment:
If someone has time to investigate how far this might be a real issue, it should be reported to jansi.